monix-reactive/shared/src/main/scala/monix/reactive/internal/builders/InputStreamObservable.scala

```diff
@@ -35,7 +35,8 @@ import scala.util.control.NonFatal
 import scala.util.{Failure, Success}
 
 private[reactive] final class InputStreamObservable(in: InputStream, chunkSize: Int) extends Observable[Array[Byte]] {
-  self =>
+
+  require(chunkSize > 0, "chunkSize > 0")
```

Comment:
Again, patterned this fix after `RangeObservable`'s `require(step != 0, "step != 0")`